pkg/cli/cli.go

```diff
@@ -423,7 +423,7 @@ func (c *CLI) getInfoFromDefaults() {
 }
 
 const unstablePluginMsg = " (plugin version is unstable, there may be an upgrade available: " +
-	"https://kubebuilder.io/migration/plugin/plugins.html)"
+	"https://kubebuilder.io/plugins/plugins-versioning)"
```

Comment:
What plugin are you using that you faced this message?
Could you please share? What plugin info is in your PROJECT file?